src/particle/pdgid/functions.py

```diff
@@ -210,6 +221,16 @@ def is_diquark(pdgid):
     return False
 
 
+def is_heavy_flavor(pdgid):
```

Comment:
See my comment before. The term "heavy flavor" is explicitly defined here, that is good, but I would still avoid it and rather teach people to use `has_charm` etc explicitly.

Reply:
This def is certainly the one that is less important to have. I can leave it out, sure.

In the end, let's be fair, we will see how users react and it will always be possible to add functionality. Though I take your point that addings lots of extra is_XXX is not going to help having a clear and clean API.

Reply:
I do see this as potentially rather handy, but I agree, let's keep the API simple at first, and see if this gets requested. The simpler building blocks are explicit and clear, and reduce the API that a user has to know to use it.

Reply:
OK, let's go with that. Thanks.